### PR TITLE
st1 [2034][FIX] stock_picking_partner_info

### PR DESCRIPTION
**Pull request by tack-yabeQuartile**
[#2034](https://www.quartile.co/web#view_type=form&model=project.task&id=2034&active_id=2034&menu_id=)

* fix to display partners info in <s>入荷 page</s> receipt form view.

### CODE REVIEW

File: stock_picking_partner_info/views/stock_picking_views.xml [hunk 1]
```diff
@@ -12,6 +12,12 @@
                     name="attrs"
                 >{'invisible': [('picking_type_code', '!=', 'outgoing')]}</attribute>
             </xpath>
+            <xpath expr="//field[@name='partner_id']" position="after">
+                <field
+                    name="partner_id"
+                    attrs="{'invisible': [('picking_type_code', '=', 'outgoing')]}"
+                />
+            </xpath>
```
Comment (yostashiro) [inline stock_picking_partner_info/views/stock_picking_views.xml]:
```.xml
            <xpath expr="//field[@name='partner_id']" position="attributes">
                <attribute
                    name="attrs"
                >{'invisible': [('picking_type_code', '=', 'outgoing')]}</attribute>
            </xpath>
            <xpath expr="//field[@name='partner_id']" position="after">
                <field
                    name="partner_id"
                    context="{'show_address': 1}"
                    options="{'always_reload': 1}"
                    attrs="{'invisible': [('picking_type_code', '!=', 'outgoing')]}"
                />
            </xpath>
```
Slightly better this way?

**Review by yostashiro — APPROVED** [review]

**Comment by tack-yabeQuartile** [comment]
I have checked GUI behavior. In receipt view I see the "Partner" w/o address, location, operation type, and cannot see the phone number.

File: stock_picking_partner_info/views/stock_picking_views.xml (hunk 1 above)
Comment (tack-yabeQuartile) [reply stock_picking_partner_info/views/stock_picking_views.xml]: I got it. I agree it is slightly easy to read.

Comment (tack-yabeQuartile) [reply stock_picking_partner_info/views/stock_picking_views.xml]: And this is a kind notation for the reader as there is another field of phone after this.

Comment (tack-yabeQuartile) [reply stock_picking_partner_info/views/stock_picking_views.xml]: I sort these fields as below to adjust layout.
1. patrner_id w/ "invisible when picking_type_code = outgoing"
2. phone after "partner_id" w/ "invisible when picking_type_code != outgoing"
3. patrner_id after "partner_id" w/ "invisible when picking_type_code != outgoing" 

This sequence give us original layout in outgoing transfer form view.

Comment (tack-yabeQuartile) [reply stock_picking_partner_info/views/stock_picking_views.xml]: I have checked the layout in both outgoing transfer and incoming transfer.

**Comment by tack-yabeQuartile** [comment]
After passed the tests, I merge this PR.

**Pull request merged**